pkg/asset-manager-utils/contracts/test/SinglePoolTestAssetManager.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma experimental ABIEncoderV2;
+
+import "../SinglePoolAssetManager.sol";
+
+pragma solidity ^0.7.0;
+
+// solhint-disable no-empty-blocks
+// solhint-disable var-name-mixedcase
+// solhint-disable private-vars-leading-underscore
```

Comment:
We seem to be out of sync with the rest of the monorepo codebase on this. A lot of the asset manager code used underscores to denote function args whereas other packages use them for private variables.

Reply:
Yeah.  Personally I like underscores for function args, I think it makes it clear what is coming from calldata and what is allocated.

This `solhint-disable` line is for something else though - it's so we can have const `ONE` and not `_ONE`

Reply:
Ah fair enough, this is probably something for another PR so we can hit both single and multi-pool AMs but we should probably move these to be more selective and just target those lines.